### ISSUE TITLE
Thumbnails are optional for the File object

### ISSUE THREAD

**Issue by kiyoto** [slack-rs/slack-rs-api#18]
A redacted example (It was a Ruby code snippet)

```js
{
    "type": "XXX",
    "file": {
        "id": "XXX",
        "created": 1461040885,
        "timestamp": 1461040885,
        "name": "XXX",
        "title": "XXX",
        "mimetype": "XXX",
        "filetype": "XXX",
        "pretty_type": "XXX",
        "user": "XXX",
        "editable": true,
        "size": 2200,
        "mode": "XXX",
        "is_external": false,
        "external_type": "",
        "is_public": true,
        "public_url_shared": false,
        "display_as_bot": false,
        "username": "",
        "url_private": "XXX",
        "url_private_download": "XXX",
        "permalink": "XXX",
        "permalink_public": "XXX",
        "edit_link": "XXX",
        "preview": "XXX",
        "preview_highlight": "XXX",
        "lines": 30,
        "lines_more": 25,
        "channels": ["C094GC4JK"],
        "groups": [],
        "ims": [],
        "comments_count": 0
    },
    "event_ts": "XXX"
}
```

**Comment by kiyoto** [comment]
@mthjones I will wait for #17 to be merged.

**Comment by mthjones** [comment]
Closed via #22.
